lib/scanner2.dart

```diff
@@ -133,50 +128,43 @@ class _Scanner2State extends State<Scanner2> {
 
   void _onCloseByUser() {
     _close();
-    _completer.completeError(PlatformException(
-        code: 'USER_CANCELED', message: 'User closed the scan window'));
+    _completer.completeError(PlatformException(code: 'USER_CANCELED', message: 'User closed the scan window'));
   }
 
   void _close() {
     if (_scanner != null) {
-      _scanner.removeFrom(document.getElementById('scanner'));
-      _scanner.stopScanning();
+      _scanner!.removeFrom(document.getElementById('scanner'));
+      _scanner!.stopScanning();
     }
-    document.getElementById('container').remove();
+    document.getElementById('container')!.remove();
   }
 
   void scannerReady() {
     window.navigator.getUserMedia(video: true).then((stream) {
-      window.navigator.mediaDevices.enumerateDevices().then((devices) {
-        _cameras =
-            devices.where((device) => device.kind == 'videoinput').toList();
-        _scanner = JsQRScanner(allowInterop(this.onQRCodeScanned),
-            allowInterop(this.provideVideo));
-        _scanner.setSnapImageMaxSize(300);
+      window.navigator.mediaDevices!.enumerateDevices().then((devices) {
+        _cameras = devices.where((device) => device.kind == 'videoinput').toList();
+        _scanner = JsQRScanner(allowInterop(this.onQRCodeScanned), allowInterop(this.provideVideo));
+        _scanner!.setSnapImageMaxSize(300);
         var scannerParentElement = document.getElementById('scanner');
-        _scanner.appendTo(scannerParentElement);
+        _scanner!.appendTo(scannerParentElement);
       }).catchError((onError) => _reject(onError));
     }).catchError((onError) => _reject(onError));
   }
 
   Promise<MediaStream> provideVideo() {
     var videoPromise;
     if (_useCamera < 0) {
-      videoPromise = getUserMedia(new UserMediaOptions(
-          video: new VideoOptions(facingMode: 'environment')));
+      videoPromise = getUserMedia(new UserMediaOptions(video: new VideoOptions(facingMode: 'environment')));
     } else {
-      videoPromise = getUserMedia(new UserMediaOptions(
-          video: new VideoOptions(
-              deviceId:
-                  new DeviceIdOptions(exact: _cameras[_useCamera].deviceId))));
+      videoPromise =
+          getUserMedia(new UserMediaOptions(video: new VideoOptions(deviceId: new DeviceIdOptions(exact: _cameras?[_useCamera].deviceId))));
```

Comment:
Looks like these lines will go beyond the default dart formatting of 80 columns.

Reply:
ya, sorry.. i was removed that default dart formatting 80 col from my vscode

Reply:
Can you change it back, at least for this workspace in vscode? Otherwise, the next time anyone else edits these files these lines will change back.

Reply:
pls check again